androidApp/src/main/kotlin/org/mifos/mobile/HomeActivity.kt

```diff
@@ -76,25 +76,34 @@ class HomeActivity : ComponentActivity() {
             val navController = rememberNavController()
 
             val appState = rememberMifosMobileState(networkMonitor = networkMonitor)
-            val darkTheme= isSystemInDarkTheme()
+
             val navDestination = when (uiState) {
                 is Success -> if ((uiState as Success).userData.isAuthenticated) {
                     PASSCODE_GRAPH
                 } else {
                     AUTH_GRAPH
                 }
-
                 else -> AUTH_GRAPH
             }
 
-            DisposableEffect(darkTheme) {
-                window?.statusBarColor = if (darkTheme) darkScrim.toArgb() else lightScrim.toArgb()
-                window?.navigationBarColor = if (darkTheme) darkScrim.toArgb() else lightScrim.toArgb()
+            val isSystemInDarkMode = isSystemInDarkTheme()
+            DisposableEffect(isSystemInDarkMode) {
+                window?.statusBarColor = if (isSystemInDarkMode) darkScrim.toArgb() else lightScrim.toArgb()
```

Comment:
Use `edgeToEdge` API instead of android `window` API's

Reply:
Used edgeToedgeAPI